tools/state-viewer/src/contract_accounts.rs

```diff
@@ -721,8 +721,7 @@ mod tests {
     /// Create a test receipt from sender to receiver with the given actions.
     fn create_receipt_with_actions(sender: &str, receiver: &str, actions: Vec<Action>) -> Receipt {
         let sender_id: AccountId = sender.parse().unwrap();
-        let signer =
-            InMemorySigner::from_seed(sender_id.clone(), near_crypto::KeyType::ED25519, "seed");
```

Comment:
this is different, the account and the seed are not the same. Please revert it - or make sure that it's ok. The latter would require testing this neard sub command so it's probably easier and safer just to leave it as is.

Reply:
fixed, thanks.